### PR TITLE
Fixed setPrototypeOf for middle aged browsers

### PR DESCRIPTION
**Pull request by wesleytodd**
As discussed in #14

### CODE REVIEW

**Comment by wesleytodd** [comment]
I manually tested this in OSx (chrome, ff, safari) and Windows (chrome, ff, IE10-11).  Works just fine as long as you have the right poly fills.  Here are the ones I included: https://github.com/wesleytodd/nighthawk/blob/master/lib/polyfills.js

I didn't test IE9 because it doesn't support the History api and has based routing is not something that I think should exist anymore.  But if someone has something that wants to support that they can go ahead and test it.

File: package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "methods": "~1.1.1",
     "parseurl": "~1.3.0",
     "path-to-regexp": "0.1.3",
+    "setprototypeof": "^1.0.0",
```
Comment (dougwilson) [inline package.json]: remove the `^` for old npm versions :)

**Comment by wesleytodd** [comment]
Fixed!!

**Comment by dougwilson** [comment]
Nice, looks like your module is exactly this: https://github.com/pillarjs/router/issues/14#issuecomment-72785898

I approve and will merge + release :)

**Comment by wesleytodd** [comment]
Yep, that is exactly what I modeled after.  It was the best and simplest I could find.

**Comment by dougwilson** [comment]
Published as 1.1.0

**Comment by wesleytodd** [comment]
Awesome, thanks!!

**Pull request closed without merging**